sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/credentials/ConfigurationClientCredentials.java

```diff
@@ -67,10 +67,10 @@ public ConfigurationClientCredentials(String connectionString)
     /**
      * Gets the base URI of the Azure App Configuration instance based on the provided connection string.
      *
-     * @return The base URI of the configuration service extracted from connection string provided.
+     * @return The String of base URI of the configuration service extracted from connection string provided.
      */
-    public URL getBaseUri() {
-        return this.credentials.baseUri();
+    public String getAppConfigurationURL() {
```

Comment:
Does this method need to be `public` if we are going to move this class into that same package as the builder and clients?

Did you ping @annelo-msft about this change?